Add note about rosetta2 in installation guide (#1271)
unit8co/darts: INSTALL.md

brew install libomp.rb
```

#### Test environment Apple M1 processor

We currently recommend to run Darts in an x_64 emulated environment on Mac computers with the Silicon M1 processor,
instead of trying to install directly with native arm64 packages, many of the dependent packages still have compatibility 
issues. The following is a proposed procedure, if you tested other procedures on similar hardware and they worked, 
please let us know about them by opening an issue or by updating this file and opening a PR. 

Before you start make sure that you have rosetta2 installed by running: 
```
pgrep oahd
``` 

> Internally rosetta is known as oah, so if you will see process id then it means that you are ready to go. 

Reviewer: ```suggestion
If you see some process id you are ready to go, as internally rosetta is known as oah.
```

> If pgrep won't return any id then install rosetta2: 

Reviewer: ```suggestion
If pgrep doesn't return any id then install rosetta2: 
```
